script.js

```diff
@@ -29,3 +29,151 @@ menuItem3.addEventListener("click", () => {
   menu.classList.remove("menu");
   menu.classList.add("hidden");
 });
+
+const portfolio = document.querySelector('#portfolio');
+
+const body = document.body;
+  const article = document.createElement('article');
+    const div0 = document.createElement('div');
+      const div1 = document.createElement('div');
+        const div2 = document.createElement('div');
+          const h3 = document.createElement('h3');
+          const i0 = document.createElement('i');
+        const img0 = document.createElement('img');
+        const p0 = document.createElement('p');
+      const ul = document.createElement('ul');
+        const li0 = document.createElement('li');
+        const li1 = document.createElement('li');
+        const li2 = document.createElement('li');
+        const li3 = document.createElement('li');
+      const div3 = document.createElement('div');
+        const button0 = document.createElement('button');
+          const p1 = document.createElement('p');
+          const a1 = document.createElement('a');
+            const i1 = document.createElement('i');
+        const button1 = document.createElement('button');
+          const p2 = document.createElement('p');
+          const a2 = document.createElement('a');
+            const img1 = document.createElement('img');
+
+
+
+let names = ['Multi-Post Stories', 'Multi-Post Stories', 'Multi-Post Stories', 'Multi-Post Stories'];
```

Comment:
As mentioned in the previous review the popup window should be styled based on the given design

### Original design
![right](https://user-images.githubusercontent.com/58376020/127572314-58e325bf-a182-4007-a07c-3db708bab56b.png)

### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/127572350-fc961697-50be-49ce-847f-8f2dce3a4b19.png)

Reply:
In none of the two previous reviews have the problem you're talking about is mentioned; the reviewers were talking about the close button not being visible in the pop-up card and the buttons not being side by side in the desktop version; as you can see, both required changes have been done, leaving the optional change of making my pop-up "more responsive". However, that is an optional change, as stated by the last reviewer, and therefore, I believe it shouldn't be a reason to reject the pull request.